file-events/src/test/groovy/net/rubygrapefruit/platform/file/HardLinkFileEventFunctionsTest.groovy

```diff
@@ -108,8 +108,9 @@ class HardLinkFileEventFunctionsTest extends AbstractFileEventFunctionsTest {
         then:
         // On Windows we seem to be getting multiple MODIFIED events
         // On Linux we _sometimes_ seem to be getting multiple events
+        // For macOS 12 we get events for both hardlink locations
```

Comment:
I wonder how/when we should check back here if this gets fixed in macOS 12.x or 13+...